whitelist.txt

```diff
@@ -6,3 +6,10 @@ pylint
 
 # Stands for 'source' package.
 src
+
+# Either a package's name or a part of it
```

Comment:
Let's write something simpler, just `# validators`. The goal of these comments is just to decypher of some words. But there are weird cases when `flake` doesn't understand very simple words like this one.

I mean there may be no sense to explain this word as a name of the package, because in some other context it can be just name of the variable or something else. Hence `# validators` looks enough here.